src/components/CertificateViewer.js

```diff
@@ -133,7 +133,7 @@ export const StatelessCertificateViewer = props => {
             Certificate issuer is <b>not</b> in the SkillsFuture Singapore
             registry for Opencerts
             <br />
-            <Link href="/faq">
+            <Link href="/faq#verifications_issuers_not_in_registry_meaning">
```

Comment:
```suggestion
            <Link href="/faq#verifications-issuers-not-in-registry-meaning">
```

For seo reasons. Underscore joins them into a single word while hyphens treats the word separately. 

https://www.ecreativeim.com/blog/index.php/2011/03/30/seo-basics-hyphen-or-underscore-for-seo-urls/

Reply:
oh yeah true, I shouldnt have used underscore thanks :)